src/compiler/compile/Component.ts

```diff
@@ -155,7 +155,7 @@ export default class Component {
 				) || { start: 0, end: 0 };
 				this.warn(svelteOptions, {
 					code: 'custom-element-no-tag',
-					message: `No custom element 'tag' option was specified. To automatically register a custom element, specify a name with a hyphen in it, e.g. <svelte:options tag="my-thing"/>. To hide this warning, use <svelte:options tag={null}/>`
+					message: 'No custom element \'tag\' option was specified. To automatically register a custom element, specify a name with a hyphen in it, e.g. <svelte:options tag="my-thing"/>. To hide this warning, use <svelte:options tag={null}/>'
```

Comment:
I don't think we want to disallow backquotes for strings. I think in some cases these were picked because it looks nicer, even though there might not be any template expressions in them.

Reply:
What if we just changed the escaped single quotes to double quotes?

Reply:
The exact text of the `message` fields of warnings is not part of the API, but I'd still much prefer to not change any of those just for linting convenience. I'd rather go with what this PR has than to adjust the actual strings.

Reply:
I raised a PR for folks to vote on this: https://github.com/sveltejs/eslint-config/pull/7